### PR TITLE
feat: store-gateway nginx path

### PR DESCRIPTION
**Pull request by cabrinha**
**What this PR does**:

Exposes a path via NGINX to reach store-gateway service.

**Checklist**
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: templates/nginx-config.yaml [hunk 1]
```diff
@@ -84,6 +84,11 @@ data:
           proxy_pass      http://{{ template "cortex.fullname" . }}-ruler.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.config.server.http_listen_port }}$request_uri;
         }
 
+        # Store Gateway
+        location ~ /store-gateway/.* {
+          proxy_pass      http://{{ template "cortex.fullname" . }}-store-gateway.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.config.server.http_listen_port }}$request_uri;
```
Comment (ShuzZzle) [inline templates/nginx-config.yaml:89]: That Service doesn't exist. Should be -headless (See: #166 ). By the way we should really have a normal ClusterIP service for this

**Review by ShuzZzle — CHANGES_REQUESTED** [review]

File: templates/nginx-config.yaml (hunk 1 above)
Comment (ShuzZzle) [reply templates/nginx-config.yaml:89]: When #169 is implemented this actually exists then

Comment (ShuzZzle) [reply templates/nginx-config.yaml:89]: Merged in #169 therefore resolved.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 * [CHANGE] Resource requests and limits have been removed from the values.yaml file. The default resources assigned will be based on the particular k8s cluster configuration, if they are not overridden.
 * [BUGFIX] Corrected structural issues found in ingress resource yaml for kube versions 1.19+
+* [ENHANCEMENT] Allow store-gateways to be reached via the NGINX ingress, allowing access to the ring, etc.
```
Comment (ShuzZzle) [inline CHANGELOG.md:7]: Holy Smokes :D thats an super old version of the chart, no? Please rebase

Comment (ShuzZzle) [reply CHANGELOG.md:7]: Also the order is incorrect and the PR number is missing to the changelog entry

**Comment by cabrinha** [comment]
I'll reopen this PR

**Pull request closed without merging**